api/collections.js

```diff
@@ -0,0 +1,95 @@
+'use strict';
+
+const QuoteCollection = require('../models/quoteCollection.js');
+const router = require('express').Router();
+const HTTPStatus = require('http-status');
```

Comment:
Maybe take a convention and stick to it. e.g. npm modules first, then blank space then your modules. also alphabetizing is a good for readability. Naming-wise you can capitalize modules and use lower case for importing single functions